R/recall_count.R

```diff
@@ -1,12 +1,14 @@
 #' count_recall_by_make
 #'
-#' @param make list string
-#' @param manufacturer A bool, with this flag we are asking for the manufacturer not the make
-#' @param start_year An integer
-#' @param end_year An integer
+#' Queries the website with a url that it generates from it's parameters.
+#'
+#' @param make List string, name(s) of the vehicle(s).
+#' @param manufacturer A boolean, with this flag we are asking for the manufacturer not the make.
```

Comment:
the API is not just for vehicles (tires and child car seats as well) - probably say `make or manufacturer name(s)`